torch/backends/cudnn/rnn.py

```diff
@@ -190,7 +196,11 @@ def forward(fn, input, hx, weight, output, hy):
         handle = cudnn.get_handle()
         fn.datatype = cudnn._typemap[input.type()]
         is_input_packed = fn.batch_sizes is not None
-
+        if is_input_packed:
+            warnings.warn(
+                "Warning: persistent algorithm not supported for variable length input."
+                "Switching to standard")
+            fn.persistent = False  # persistent algo is not supported for variable length input
```

Comment:
It's more like standard algo has unpredictable perf because with each new release cublas heuristics can change, and standard algo uses cublas.